mypy.ini

```diff
@@ -3,4 +3,7 @@ mypy_path = src
 explicit_package_bases = True
 namespace_packages = True
 show_error_codes = True
-strict = True
\ No newline at end of file
+strict = True
+
+[mypy-dateutil.*]
+ignore_missing_imports = True
```

Comment:
Since `types-python-dateutil` is in `requirements-dev.txt`, can't we drop this ignore?